frontend/src/components/classify-failures.js

```diff
@@ -73,39 +70,18 @@ export class ClassifyFailuresTable extends React.Component {
     this.getResultsForTable();
   }
 
-  onExceptionToggle = isExpanded => {
-    this.setState({isExceptionOpen: isExpanded}, this.applyFilter);
-  }
-
-  applyExceptionFilter = () => {
-    let { filters, exceptionSelections } = this.state;
-    if (exceptionSelections.length > 0) {
-      filters["metadata.exception_name"] = Object.assign({op: 'in', val: exceptionSelections.join(';')});
+  applyFilter = (field, value) => {
```

Comment:
I'm looking at updating the existing updateFilters function on this class to use instead of this.